libcloud/compute/drivers/softlayer.py

```diff
@@ -306,6 +308,10 @@ def create_node(self, **kwargs):
         if kwargs.get('ex_local_disk') is False:
             local_disk = 'false'
 
+        post_uri = ''
```

Comment:
can post_uri be submitted with an empty value? Are there any tests to support this?